diff --git a/app/__init__.py b/app/__init__.py
index 0fdc86f8b300..598a9b3f3902 100644
--- a/app/__init__.py
+++ b/app/__init__.py
@@ -137,6 +137,7 @@ for cb_dir in cache_busting_dirs:
 
         # Create mapping for relative path urls
         map_path = build_path.replace(app.config['APP_ROOT'], '')
+        map_path = build_path.replace(app.config['CONFIG_PATH'], '')
         if map_path.startswith('/'):
             map_path = map_path[1:]
         app.config['CACHE_BUSTING_MAP'][cb_file] = map_path
